components/webpack/webpack.config.js

```diff
@@ -11,13 +11,14 @@ module.exports = (env, argv) => ({
   resolve: {
     extensions: ['.js', '.tsx', '.ts', '.json'],
     alias: {
-      'bittorrent-tracker': path.resolve(__dirname, '../../node_modules/bittorrent-tracker'),
       'brave-ui': path.resolve(__dirname, '../../node_modules/brave-ui/src'),
-      'dgram': 'chrome-dgram',
-      'dns': path.resolve(__dirname, '../common/dns.ts'),
-      'net': 'chrome-net',
-      'torrent-discovery': path.resolve(__dirname, '../../node_modules/torrent-discovery')
```

Comment:
- No more need to depend on `bittorrent-tracker` or `torrent-discovery` fork.
- No more need to specify `net` to `chrome-net` or `dgram` to `chrome-dgram` alias.